components/IscnUploadForm.vue

```diff
@@ -985,6 +987,7 @@ export default class IscnUploadForm extends Vue {
       return
     } finally {
       this.uploadStatus = '';
+      this.fetchWalletBalance()
```

Comment:
Maybe replace `getAccountBalance` call with `fetchWalletBalance` and `getBalance` from store?